deployment_steps_cdk.adoc

```diff
@@ -1 +1,35 @@
-Deployment takes about {deployment_time} to complete.
\ No newline at end of file
+To deploy {partner-product-short-name} on AWS, complete the following steps. The stack takes about {deployment_time} to launch.
+
+. Confirm that you have the https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites[prerequisites^] to install the AWS CDK.
+. Install the AWS CDK.
+. Clone the {partner-product-short-name} Quick Start repository:
+```
+git clone https://github.com/aws-quickstart/quickstart-{partner-product-name}.git
```

Comment:
this will need to be more dynamic, like this, so that it builds correctly (orgs can differ, product name doesn't always match repo name:
https://github.com/{quickstart-github-org}/{quickstart-project-name}.git